h2o-core/src/main/java/water/Scope.java

```diff
@@ -92,4 +89,7 @@ static public void untrack(Iterable<Key<Vec>> keys) {
     HashSet<Key> xkeys = scope._keys.peek();
     for (Key<Vec> key : keys) xkeys.remove(key); // Untrack key
   }
+
+  @Override
```

Comment:
I agree with Michal. Scope is conceptually not Closeable, user should not be manipulating with an instance.
Exposing instance which does not mean anything is really misleading. try(Closeable){} does not even save the much, just a little bit of a syntactic sugar.